samples/next-ab-testing/pages/_app.js

```diff
@@ -0,0 +1,32 @@
+import "../styles/global.css";
+import Head from "next/head";
+
+import { emitter, EXPERIMENT_PLAYED, EXPERIMENT_WON } from "next-experiments";
+
+if (typeof window !== "undefined") {
```

Comment:
Is it the same hooks you mentioned on Jira?

Reply:
I think they would be different!

In jira, I meant React hooks specifically, so that we can use experiments in code.

Here, I meant more maybe as a callback function or value passed by the `Experiment` parent as a render prop or a context value that the `Variant` can use.

Nothing blocking this PR though, we can improve on it later!

ex: 

```
// pseudo code

<Experiment name="experiment" defaultVariantName="a">
  <Variant name="a">
    {(win, hasPlayed) => <button onClick={() => win()}>Variant A - {hasPlayed ?? 'do something after I was played!'}</button>}
  </Variant>
  ...
</Experiment>

```